app/services/ping.rb

```diff
@@ -12,13 +12,22 @@ def ping
     end
 
     def statuses
+      plugins = Foreman::Plugin.all.map do |plugin|
+        {
+          id: plugin.id,
+          name: plugin.name,
+          version: plugin.version,
+          author: plugin.author,
+          description: plugin.description,
```

Comment:
Curious your thoughts on why send out author and description on a status API?

Reply:
it was there before

https://github.com/theforeman/foreman/blob/da7bca25f644a31f0f7b45194cc5fc6102c41cc3/app/registries/foreman/plugin.rb#L233-L235

and I didn't want to loose that data.

*personally* I only need `id` and `version` :)

Reply:
Speaking of name… the `Foreman::Plugin` object has both, `name` and `id` attributes, but both seem to be the same?

Reply:
Hm.. what could be id of the plugin except its name anyway?.. AFAIK it's not possible to install two versions of a plugin and use both at the same time. I'd expect that id == name.

But apparently there can be such thing as id != name:
https://github.com/theforeman/foreman/blob/036bbe66b1acf7f6e1a3ffade702f90dd85ec7fe/app/registries/foreman/plugin.rb#L96-L98

Reply:
Yeah, name is the name of the gem, while id is what you put in when you register the plugin, so I'd actually say the id is the name and the name is worthless 😅

Reply:
That said, let's make the data `{name: plugin.id, version: plugin.version}` and be done with it?

Reply:
@evgeni +1